### PR TITLE
EWB-3478 Expose service metadata to clients

### PR DESCRIPTION
**Pull request by vincewhite**
# Description

Adds getMetadata() gRPC calls to Customer/Diagram/Network consumer clients.

# Associated tasks

Relies on:
https://github.com/zepben/evolve-grpc/pull/80

evolve-sdk-jvm:
https://github.com/zepben/evolve-sdk-jvm/pull/147

# Checklist

If any of these are not applicable, strikethrough the line `~~like this~~`. **Do not delete it!**. Let the reviewer decide if you should have done it.

### Code
- [x] I have performed a self review of my own code (including checking issues raised when creating the PR).
- [x] I have added/updated unit tests for these changes, and if not I have explained why they are not necessary.
- [x] I have commented my code in any hard-to-understand or hacky areas.
- [x] I have handled all new warnings generated by the compiler or IDE.
- [x] I have rebased onto the target branch (usually main).
      
### Documentation
- [x] I have updated the changelog.
- [x] I have updated any documentation required for these changes.

# Breaking Changes
- [x] I have considered if this is a breaking change and will communicate it with other team members if so.

Please leave a summary of the breaking changes here. This is useful for the reviewer, but also is useful for communication to the team when merged (e.g. you could copy and paste the summary into slack).

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [EWB-3478 Update Python SDK](https://app.clickup.com/t/6929263/EWB-3478)

**Comment by ryanjcoleman** [comment]
Task linked: [EWB-3478 Update Python SDK](https://app.clickup.com/t/6929263/EWB-3478)

File: src/zepben/evolve/streaming/get/network_consumer.py [hunk 1]
```diff
@@ -24,7 +27,7 @@
     PowerElectronicsConnectionPhase, BatteryUnit, PhotoVoltaicUnit, PowerElectronicsWindUnit, BusbarSection, LoadBreakSwitch, TransformerTankInfo, \
     TransformerEndInfo, TransformerStarImpedance, EquipmentContainer, NetworkHierarchy, MultiObjectResult, CimConsumerClient, NoLoadTest, OpenCircuitTest, \
     ShortCircuitTest, EquivalentBranch, ShuntCompensatorInfo, LvFeeder, CurrentRelay, CurrentTransformer, CurrentRelayInfo, SwitchInfo, \
-    CurrentTransformerInfo, EvChargingUnit, RegulatingControl, TapChangerControl
+    CurrentTransformerInfo, EvChargingUnit, TapChangerControl, service_info
```
[the remark this answers is not in the document]
Comment (kgreav) [reply src/zepben/evolve/streaming/get/network_consumer.py:30]: yep makes sense as it's an abstract class - good catch.

File: src/zepben/evolve/services/common/meta/metadata_translations.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#  Copyright 2023 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from google.protobuf.timestamp_pb2 import Timestamp as PBTimestamp
+from zepben.protobuf.metadata.metadata_data_pb2 import DataSource as PBDataSource
+from zepben.protobuf.metadata.metadata_data_pb2 import ServiceInfo as PBServiceInfo
+
+from zepben.evolve import DataSource, service_info
```
[the remark this answers is not in the document]
Comment (kgreav) [reply src/zepben/evolve/services/common/meta/metadata_translations.py:10]: yes

File: src/zepben/evolve/services/common/meta/service_info.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+from dataclassy import dataclass
+from typing import TYPE_CHECKING, List
+if TYPE_CHECKING:
+    from zepben.evolve import DataSource
+
+__all__ = ["service_info"]
+
+
+@dataclass(slots=True)
+class service_info(object):
```
[the remark this answers is not in the document]
Comment (Lax125) [reply src/zepben/evolve/services/common/meta/service_info.py]: I renamed it to `ServiceInfo`.

**Review by Lax125 — APPROVED** [review]
LGTM

**Pull request merged**